health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/Map.js

```
@@ -0,0 +1,400 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { scaleQuantile } from "d3-scale";
+import ReactTooltip from "react-tooltip";
+import { ZoomableGroup, Geographies, ComposableMap, Geography } from "react-simple-maps";
+// import { Icon } from "../../common/Icon";
+import { getTitleHeading } from "../../../utils/locale";
+import BoundaryTypes from "../../../utils/enums";
+const Map = ({
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)